src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/arg_max_min/arg_max_min_kernel_axis.cpp

```diff
@@ -117,6 +122,9 @@ JitConstants ArgMaxMinKernelAxis::GetJitConstants(const arg_max_min_params& para
 
     if (params.outputs_num == 2) {
         jit.AddConstant(MakeJitConstant("SECOND_OUTPUT_EXIST", 1));
+        if (params.outputs.size() > 1) {
+            jit.AddConstant(MakeJitConstant("MULTIPLE_OUTPUTS", 1));
```

Comment:
I think these flags are to be confusing in the future. 

please use common identifier function here too.

Reply:
done